fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/CommonDatabaseAdapter.java

```diff
@@ -424,14 +424,14 @@ public void runStatement(IDatabaseStatement stmt) {
     public void createSequence(String schemaName, String sequenceName, int cache) {
         /*
          * <CODE>CREATE SEQUENCE fhir_sequence
-             AS BIGINT
-     START WITH 1000
-          CACHE 1000
-       NO CYCLE;</CODE>
-    */
+         * AS BIGINT
+         * START WITH 20000
```

Comment:
why 20000 and not 10000?

Reply:
plenty of space then, for any expansions we need based on sequences

Reply:
ok, was just curious